### PR TITLE
Proposal for SAF change message

### PR DESCRIPTION
**Pull request by Lineflyer**
The corresponding FAQ entry is not yet existing.
Will take care of that next days (before beta release might occur).

I wanted to keep it as short as possible but
- Provide reason
- Advise to use wizard
- Link to more info (for those interested)

### CODE REVIEW

File: main/res/values/changelog_master.xml [hunk 1]
```diff
@@ -4,7 +4,12 @@
   <string name="changelog_master" translatable="false"><Data><![CDATA[<div>
 <h1>Nightly build:</h1>
 
-*** TODO: Need some introductionary text about the SAF change ***
+<h2>
+We had to do extensive changes to the way c:geo gets access to your file system (e.g. for storing data, reading offline maps) to be prepared for requirements on new Android versions.
```
Comment (eddiemuc) [inline main/res/values/changelog_master.xml:8]: Reformulation suggestion:
Starting with Android 11, Google enforces usage of <a href="https://developer.android.com/about/versions/11/privacy/storage">Scoped Storage</a> to access public file system on mobile devices. 
C:geo uses this file system to store and access data like e.g. offline maps, GPX files, backups and other data.  Major changes were done to comply to the new access system.  
After upgrading to this c:geo version some user action is necessary to allow c:geo accessing necessary public folders. A wizard will guide you through the needed changes.  Please follow these steps in the wizard carefully!

Comment (Lineflyer) [reply main/res/values/changelog_master.xml:8]: To long and to much technology terms IMHO

Comment (eddiemuc) [reply main/res/values/changelog_master.xml:8]: Ok, another try:
To comply to new Android versions, changes where necessary to the way c:geo accesses the file system (e.g. for storing data, reading offline maps). As a consequence you will need to grant folder access permissions to c:geo. 
A wizard will guide you through the needed changes. Please follow the steps in the wizard carefully!

Comment (fm-sys) [reply main/res/values/changelog_master.xml:8]: This wording would not be that personal. While the app content should be focused to the user side, a changelog is a summary of what the team has done for c:geo IMHO. Therefore I like "we" and @Lineflyer's version in total.

Comment (eddiemuc) [reply main/res/values/changelog_master.xml:8]: Go for it

**Review by eddiemuc — APPROVED** [review]

File: main/res/values/changelog_master.xml (hunk 1 above)
Comment (Lineflyer) [reply main/res/values/changelog_master.xml:8]: I guess that >80% of people will anyhow skip the screen without reading...I will also place this sentence into the PlayStore changelog...but still. Don't expect many people reading it.

So lets merge and thanks for your comments.

**Pull request merged**